docs/user/plugins/cloudFoundry/cloud-foundry-plugin.md

```diff
@@ -0,0 +1,96 @@
+# Cloud Foundry Plugin
+
+## Introduction
+The Cloud Foundry Plugin transforms a CSAR to an artifact, that can be deployed on the Cloud Foundry Platform. In general the following steps have to be done to transform and deploy a CSAR:
+
+1. upload CSAR with the GUI/ CLI
+2. select Platform CloudFoundry
+3. enter required Inputs
+4. start Transformation
+5. download artifact
+6. unzip at target location
+7. start deploying with with execution of deploy.sh found in output/scripts/ in target artifact
+
+Before the Transformation can start the Cloud Foundry credentials and additional properties have to be set:
+
+- `CF_Instance_Username`: Username of CloudFoundry user account
+- `CF_Instance_Api`: The endpoint of the provider
+- `CF_Instance_Organization`: The organization of the user account which should be used to deploy
+- `CF_Instance_Password`: Password of CloudFoundry user account
+- `CF_Instance_Space`: The space of the user account which should be used to deploy
+
+The credentials are used to get a list of services which can be used for deployment.
+For more details on supported NoteTypes, basics of Cloud Foundry, the Transformation and created scripts, see the documents in `dev/plugins/cloudFoundry`.
+
+## Artifact
+The Artifact is the result of the transformation which contains all necessary files which are needed to deploy an application on Cloud Foundry. After you open the Artifact you can see the log of the Transformation, the app folder(s) which contain your application and the output folder which contains the files for deployment. If you provide multiple applications in your CSAR the artifact will contain multiple app folders called app1, app2 and so on.
```

Comment:
Not sure if enough context is provided for the term `application`. I know what you mean ofc-- maybe you could link to a section which explains what the plugin means exactly with that term,